actors/verifreg/src/lib.rs

```diff
@@ -762,6 +811,133 @@ where
     )
 }
 
+// Deserializes and validates a token receiver hook payload.
+fn validate_tokens_received(
+    raw: &RawBytes,
+    my_id: u64,
+) -> Result<TokensReceivedParams, ActorError> {
+    let payload: TokensReceivedParams = deserialize(raw, "receiver hook payload")?;
```

Comment:
Can you explain why you do this here instead of [here](https://github.com/filecoin-project/builtin-actors/pull/601/files#diff-d3ee10a93cdca54a99f37a1b78a40c6b34894b097776d97030cf360eaaf165ecR1007).  It seems to be breaking the convention with no benefit.

Reply:
I've reverted to the pattern.